### PR TITLE
[FS-2035] :  Modify  XCode project

### PR DESCRIPTION
**Pull request by bhushankini**
Use plutil to make changes to Project

### CODE REVIEW

File: packages/flagship/src/lib/ios.ts [hunk 1]
```diff
@@ -21,11 +24,8 @@ export function bundleId(configuration: Config): void {
 
   helpers.logInfo(`updating iOS bundle id`);
 
-  fs.update(
-    path.ios.infoPlistPath(configuration),
-    /<key>CFBundleIdentifier<\/key>\s+<string>[^<]+<\/string>/,
-    `<key>CFBundleIdentifier</key><string>${bundleId}</string>`
-  );
+  exec(`plutil -replace CFBundleIdentifier -string ${bundleId} ${path.ios.infoPlistPath(configuration)}`);
```
Comment (skyeckstrom) [inline packages/flagship/src/lib/ios.ts:27]: Instead of using `execSync` and passing unsantizied inputs, we should use `execFileSync` so the `bundleId` can be escaped properly. The risk with this approach is a malicious Flagship project could pass in shell code in the bundleId that would be blindly executed.

Change this to something like
```typescript
execFileSync('plutil', ['-replace', 'CFBundleIdentifier', '-string', bundleId, path.ios.infoPlistPath(configuration)])
```

File: packages/pirateship/env/common.js [hunk 2]
```diff
@@ -46,5 +46,6 @@ module.exports = {
     ios: 'com.brandingbrand.reactnative.pirateship',
   },
   associatedDomains: [],
-  targetedDevices: 'Universal'
+  targetedDevices: 'Universal',
+  UIBackgroundModes: ['remote-notification']
```
Comment (skyeckstrom) [inline packages/pirateship/env/common.js:50]: Can you remove this unrelated change?

**Review by skyeckstrom — CHANGES_REQUESTED** [review]

File: packages/flagship/src/lib/ios.ts (hunk 1 above)
Comment (bhushankini) [reply packages/flagship/src/lib/ios.ts]: done

File: packages/pirateship/env/common.js (hunk 2 above)
Comment (bhushankini) [reply packages/pirateship/env/common.js]: done
